proto/frequenz/api/dispatch/dispatch.proto

```diff
@@ -127,7 +99,7 @@ message DispatchFilter {
   repeated uint64 microgrid_ids = 2;
 
   // Filter by dispatch type
-  repeated DispatchType types = 3;
+  repeated string types = 3;
```

Comment:
I'd suggest adding documentation about what it is meant for, now that the enum is being removed.